chef/cookbooks/network/templates/default/ovs-pre-up.sh.erb

```diff
@@ -1,6 +1,7 @@
 #! /bin/bash
 
 ovs-vsctl br-exists <%= @bridgename %> || exit 0
+ovs-vsctl set bridge <%= @bridgename %> other-config:forward-bpdu=<%= @ovs_forward_bpdu %>
 <%
```

Comment:
Just found out that setting this here will have no effect if neutron has https://github.com/openstack/neutron/commit/1f8378e0ac4b8c3fc4670144e6efc51940d796ad because the ovs agent overwrites other_config when it starts.

Reply:
> Just found out that setting this here will have no effect if neutron has [openstack/neutron@1f8378e](https://github.com/openstack/neutron/commit/1f8378e0ac4b8c3fc4670144e6efc51940d796ad) because the ovs agent overwrites other_config when it starts.

@djoreilly Did you test this with something newer than newton? I thought the above problem is newton specific (and since yesterday fixed in our packages)

Reply:
I had only tested on newton. I have just tested pike and there is no problem.

Reply:
this problem should be fixed though